### PR TITLE
Rubygems preparation

### PR DESCRIPTION
**Pull request by portellaa**
* Updated gemspec with repo and authors
* Updated version

### CODE REVIEW

File: fastlane-plugin-cosigner/fastlane-plugin-cosigner.gemspec [hunk 1]
```diff
@@ -6,21 +6,18 @@ require 'fastlane/plugin/cosigner/version'
...
   spec.name          = 'fastlane-plugin-cosigner'
   spec.version       = Fastlane::Cosigner::VERSION
-  spec.author        = %q{Luís Portela Afonso}
-  spec.email         = %q{luis.afonso@mindera.com}
+  spec.author        = %q{André Pacheco Neves, Luís Portela Afonso}
+  spec.email         = %q{ios@mindera.com}
 
   spec.summary       = %q{A fastlane plugin to help you sign your iOS builds}
-  # spec.homepage      = "https://github.com/<GITHUB_USERNAME>/fastlane-plugin-cosigner"
+  spec.homepage      = "https://github.com/mindera/fastlane-plugin-cosigner"
```
Comment (p4checo) [inline fastlane-plugin-cosigner/fastlane-plugin-cosigner.gemspec]: Shouldn't Mindera have uppercase "m"? 🚓 

(the link works the same, btw 😛)

**Review by p4checo — APPROVED** [review]

File: fastlane-plugin-cosigner/fastlane-plugin-cosigner.gemspec (hunk 1 above)
Comment (portellaa) [reply fastlane-plugin-cosigner/fastlane-plugin-cosigner.gemspec]: 🙄

**Comment by p4checo** [comment]
You make me proud 😊

**Comment by portellaa** [comment]
😊

**Pull request merged**